pkg/server/status/recorder.go

```diff
@@ -625,8 +625,9 @@ type registryRecorder struct {
 func extractValue(name string, mtr interface{}, fn func(string, float64)) error {
 	switch mtr := mtr.(type) {
 	case metric.WindowedHistogram:
+		// Use cumulative count here
+		fn(name+"-count", float64(mtr.(*metric.Histogram).TotalCount()))
```

Comment:
Is this type assertion always possible? After all, this interface is implemented in a few places:

![Uploading image.png…]()

Reply:
Apparently not, thanks for pointing it out. Will think harder about this! I was hoping to get this one fixed before pushing out the bigger refactor in https://github.com/cockroachdb/cockroach/issues/98621.